models/marts/ledger_current_state/claimable_balances_current.sql

```diff
@@ -0,0 +1,63 @@
+{{
+    config(
+        tags = ["current_state"],
+        materialized = "incremental",
+        unique_key = "balance_id"
+    )
+}}
+/* Returns the latest state of each claimable balance in the `claimable_balances` table.
+
+    Rank all rows for a claimable balance by closed_at timestamp and pick the latest one.*/
+
+with
+    current_balance as (
+        select
+            cb.balance_id
+            , cb.claimants
+            , cb.asset_type
+            , cb.asset_code
+            , cb.asset_issuer
+            , cb.asset_amount
+            , cb.sponsor
+            , cb.flags
+            , cb.last_modified_ledger
+            , cb.ledger_entry_change
+            , cb.deleted
+            , cb.batch_id
+            , cb.batch_run_date
+            , cb.batch_insert_ts
+            , cb.closed_at
+            , cb.ledger_sequence
+            , row_number()
+                over (
+                    partition by cb.balance_id
+                    order by cb.closed_at desc
+                ) as rn
+        from {{ ref('stg_claimable_balances') }} as cb
+        {% if is_incremental() %}
+            -- limit the number of partitions fetched incrementally
+            where
+                cb.batch_run_date >= timestamp_sub(current_timestamp(), interval 7 day)
```

Comment:
Does this cause a datetime vs timestamp error?

Reply:
Good catch. Addressed in https://github.com/stellar/stellar-dbt-public/pull/66/commits/d7e567e22d1627c6feb682298733216bfd8c072f